src/app/components/content-finder-page/content-finder-page.component.spec.ts

```diff
@@ -96,7 +95,7 @@ describe('ContentFinderPageComponent', () => {
   it('Should reset page to empty string Title, "Code" string selFormat, and empty array', () => {
     component.reset();
     expect(component.title).toEqual('');
-    expect(component.selFormat).toEqual('Code');
+    expect(component.selFormat).toEqual(['Code','Document', 'Powerpoint']);
```

Comment:
Just to confirm, this means that during the test, after the component is reset, all 3 checkboxes are checked?

Reply:
Correct.  The default start is all selected.